draft-ietf-quic-transport.md

```diff
@@ -380,13 +380,13 @@ header contains the following fields:
 
 Header Form:
 
-: The most significant bit (0x80) of the first octet is set to 1 for long
-  headers and 0 for short headers.
+: The most significant bit (0x80) of octet 0 (the first octet) is set to 1 for
+  long headers.
 
 Long Packet Type:
 
-: The remaining seven bits of first octet of a long packet is the packet type.
-  This field can indicate one of 128 packet types.  The types specified for this
+: The remaining seven bits of octet 0 of a long packet is the packet type.  This
```

Comment:
I think "of a long packet" is redundant given "Long Packet Type:" is the title as well, and it makes the sentence a bit awkward.